app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -65,27 +69,38 @@
 }
 
 .gem-c-feedback__prompt-question {
-  display: inline-block;
+  vertical-align: text-top;
 
   // There's a global h3 rule in some layouts that interferes with this component
   margin: 0;
 
   margin-left: govuk-spacing(4);
   margin-right: govuk-spacing(4);
+  margin-top: govuk-spacing(2);
+  margin-bottom: govuk-spacing(2);
 
   &:focus {
     outline: 0;
   }
 
+  @include govuk-media-query($from: mobile) {
+    display: inline-block;
+  }
```

Comment:
I know this is a draft PR, but I think this change doesn't make any difference? It's saying from mobile onwards make it inline-block, but line 68 previously said that, it just wasn't wrapped in a media query. Unless you're going to do something specific below 320px wide?

Reply:
I don't disagree, however I was testing against small screen sizes on chrome and below 320, without this, it looks pretty rubbish. The buttons are squashed together. I normally wouldn't worry about below 320 but chrome now has a built in view for the galaxy fold, which is apparently only 280px. I can't back this up with any know-how about if this phone is even worth considering. This is something for me to check with the frontend community probably.

Reply:
Matt Hobbs sent me this https://docs.google.com/spreadsheets/d/1g2cZ_J-gAHeXtXVFWwj9TjYOSqEDVKPCfLEWzDR_Oq4/edit#gid=1399476044 Top 500 stats for screen sizes over the past month. I don't see any screen widths of lower than 320px. With this in mind I'm happy to just move this into the default for the class. It would still be technically broken below around 300px but we don't really have a need to support below that. What do you reckon?

Reply:
I'm slightly confused - before this change, below 320px, the Yes/No buttons move onto separate lines. Don't they still do that even after being restyled to look like buttons?

Reply:
I re-tested this and for this element specifically it doesn't make a difference 🤦  Will amend and push.